ui-components/src/overlays/Modal.docs.mdx

```diff
@@ -0,0 +1,27 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs"
+import { Modal } from "./Modal"
+
+# Modal
+
+The modal component displays a modal with a title and optional children typed as either a string or JSX. The modal includes a number of other optional settings.
+
+## Component Properties
+
+<ArgsTable of={Modal} />
+
+You can apply CSS variables to the `.modal` selector to customize the appearance of the component.
+
+| Name                        | Type               | Description                     | Default                                                   |
+| --------------------------- | ------------------ | ------------------------------- | --------------------------------------------------------- |
+| `--title-color`             | Color              | Title text color                | <Swatch colorVar="--bloom-color-gray-900" />              |
+| `--title-font-size`         | Size               | Title text size                 | `--bloom-font-size-base-alt`                              |
+| `--title-font-family`       | Font               | Title font family               | `--bloom-font-sans`                                       |
+| `--content-font-color`      | Color              | Title text color                | <Swatch colorVar="--bloom-color-gray-700" />              |
+| `--max-width`               | Size               | Modal max width                 | `--bloom-width-5xl`                                       |
+| `--desktop-min-width`       | Size               | Modal min width on desktop      | `--bloom-width-lg`                                        |
+| `--footer-background-color` | Color              | Footer background color         | <Swatch colorVar="--bloom-color-primary-lighter" />       |
+| `--border-radius`           | Size               | Border radius of modal, content | `--bloom-rounded`                                         |
+| `--background-color`        | Color              | Border radius of modal, content | <Swatch colorVar="--bloom-color-white" />                 |
```

Comment:
It looks like all the descriptions here on to the bottom need to be updated.

Reply:
Fixed 👍